packages/metals-vscode/src/util.ts

```diff
@@ -115,6 +115,13 @@ export function getJavaVersionFromConfig() {
   return undefined;
 }
 
+export function getJavaVersionOverride(): string | undefined {
+  return workspace
+    .getConfiguration("metals")
+    .get<string>(UserConfiguration.MetalsJavaHome)
+    ?.trim();
```

Comment:
Maybe if empty string you want to return `undefined`?

Reply:
Done!